MathLib/mechanics_invariants.h

```diff
@@ -0,0 +1,166 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+#ifndef MATHLIB_MECHANICS_INVARIANTS_H
+#define MATHLIB_MECHANICS_INVARIANTS_H
```

Comment:
Maybe rename it to `MathLib/KelvinVectorBusiness.`? :smile:
The file does not only cover invariants. (It computes the inverse, which is not an invariant I guess)